pydsfapi/src/pydsfapi/__init__.py

```diff
@@ -0,0 +1,4 @@
+SOCKET_DIRECTORY = "/var/run/dsf"
```

Comment:
Please change to `/run/dsf`. Although DSF actually still uses `/var/run/dsf` the folder `/var/run` has been deprecated by systemd and `/run` should be used instead.

Reply:
This change is just copied over from another file.
I'll change it - I'm all in favour of adopting the new system layout!